src/front/glsl_new/error.rs

```diff
@@ -0,0 +1,44 @@
+use std::fmt;
```

Comment:
nit: let's use the include style of other modules, i.e. one statement per top-level use. In this case, `use std::{fmt, io}`

Reply:
Done